addons/packages/multus-cni/3.8.0/test/README.md

```diff
@@ -0,0 +1,9 @@
+# MULTUS-CNI E2E TESTS
+
+## USAGE
```

Comment:
I was able to run the test and it passed. However, there was a bit of trial and error to get the test to pass. I first needed to install a package repository that contained the 3.8.0 package so that it could be installed by the test. Second, I had to create an empty `test/e2e/multus-cni-values.yaml` file since the package install in the test is looking for that.

Could you update the USAGE section with these and/or other additional steps that a user would need to take in order to have a successful test run.

Reply:
Good point, @seemiller . I added a prerequisites section into test readme. If you run make e2e-test from test directory, the test cases will find values file from e2e/multihomed-testfiles/multus-cni-values.yaml